src/DataSource/Loader/SftpLoader.php

```diff
@@ -21,8 +21,8 @@
 use League\Flysystem\PhpseclibV3\SftpAdapter;
 use League\Flysystem\PhpseclibV3\SftpConnectionProvider;
 use Pimcore\Bundle\DataImporterBundle\Exception\InvalidConfigurationException;
-use Pimcore\File;
 use Pimcore\Logger;
+use Symfony\Component;
```

Comment:
```suggestion
use Symfony\Component\Filesystem\Filesystem;
```

Reply:
that would lead to a douplicate namespace ...